### PR TITLE
block/ladder.go: Change Direction to Face to allow states 0 and 1

### PR DESCRIPTION
**Pull request by Gewinum**
Fixed a bug where ladder wasn't being added to creative inventory. It was happening due to ladder in dragonfly not allowing facing_direction to be 0 and 1. We already discussed this in Discord:
image

### CODE REVIEW

File: server/block/ladder.go [hunk 1]
```diff
@@ -86,17 +86,17 @@ func (l Ladder) EncodeItem() (name string, meta int16) {
...
 // Model ...
 func (l Ladder) Model() world.BlockModel {
-	return model.Ladder{Facing: l.Facing}
+	return model.Ladder{Facing: cube.Direction(l.Facing)}
 }
 
 // allLadders ...
 func allLadders() (b []world.Block) {
-	for i := cube.Direction(0); i <= 3; i++ {
+	for i := cube.Face(0); i <= 5; i++ {
```
Comment (Gewinum) [inline server/block/ladder.go:99]: @TwistedAsylumMC shall I use cube.Faces() instead of this?

Comment (TwistedAsylumMC) [reply server/block/ladder.go:99]: Yep

File: server/block/ladder.go [hunk 2]
```diff
@@ -17,12 +17,12 @@ type Ladder struct {
 	sourceWaterDisplacer
 
 	// Facing is the side of the block the ladder is currently attached to.
```
Comment (TwistedAsylumMC) [inline server/block/ladder.go:19]: I'd probably leave a comment saying cube.FaceUp and cube.FaceDown do not do anything. Do just check in game first that it is the case

Comment (Gewinum) [reply server/block/ladder.go:19]: in game the block appears but it is invisible and somehow you can still climb it

**Comment by Gewinum** [comment]
wait ive just noticed a typo in my commit name... why did i write commend instead of commend.....

**Comment by Gewinum** [comment]
womp womp

**Comment by Gewinum** [comment]
I went to the aquarium this weekend, but I didn’t stay long. There’s something fishy about that place.

**Pull request merged**